include/igraph_error.h

```diff
@@ -564,7 +564,10 @@ DECLDIR int IGRAPH_FINALLY_STACK_SIZE(void);
  */
 
 #define IGRAPH_FINALLY(func,ptr) \
-    IGRAPH_FINALLY_REAL((igraph_finally_func_t*)(func), (ptr))
+    { \
+        if (0) { func(ptr); } \
```

Comment:
Can we add a comment here explaining what this hack is for (in case we won't remember 20 years later)?

Reply:
Sure, it's done now :)